charts/merlin/templates/merlin-deployment.yaml

```diff
@@ -165,6 +165,8 @@ spec:
           - name: FEAST_BIG_TABLE_CONFIG
             value: {{ .Values.merlin.transformer.feast.bigtableStorage | toJson | quote }}
           {{- end }}
+          - name: FEAST_BIGTABLE_CREDENTIAL
+            value: {{ .Values.merlin.transformer.feast.bigtableCredential}}
```

Comment:
tried that before but inferenceservice could not mount volume contains that secret. Hence pass this to env var

Reply:
I think it's possible to do that, but instead of mounting the secret, we need to mount the service account that is linked to that secret. Something like this: https://kserve.github.io/website/0.7/modelserving/storage/s3/s3/

Anyway, your implementation is lgtm.

Reply:
I've tried something similar like this https://github.com/kserve/kserve/issues/766, but the pod still use compute service account instead of the one that specified in inferenceservice manifest